translib/transformer/xfmr_intf.go

```diff
@@ -238,6 +261,31 @@ func performIfNameKeyXfmrOp(inParams *XfmrParams, requestUriPath *string, ifName
 	return err
```

Comment:
line 244: Do we need a case of ifType == IntfTypePortChannel ?

Reply:
Added check for portchannel in case UPDATE/REPLACE for non-existent portchannel interface. 
Create is OK to proceed without check.